boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -0,0 +1,253 @@
+.. _Intel_S1000:
+
+Intel S1000
+###########
+
+Overview
+********
+
+The Intel S1000 ASIC is designed for complex far-field signal processing
+algorithms that use high dimensional microphone arrays to do beamforming,
+cancel echoes, and reduce noise. It connects to a host processor chip via
+simple SPI and I2S interfaces, to the microphone array via I2S or PDM
+interfaces, and to speakers via I2S. In addition, it has an I2C interface
+for controlling platform components such as ADCs, DACs, CODECs and PMICs.
+
+Intel S1000 is designed to handle speech recognition at the edge (e.g.
+locally) so some of your voice commands should still work when not connected
+to the network.[1]_
+
+The features include the following:
+
+- Digital Signal Processors
+
+  - Dual Tensilica LX6 cores @ 400 MHz with HiFi3 DSP
+  - Single precision scalar floating-point instructions
+  - 16KB 4-way I$; 48KB 4-way D$
+  - Up to 2400 DMIPS, 3.2 GMACS (16x16), 800 MFLOPS of Compute
+
+- Speech Accelerators
+
+  - A GMM (Gaussian Mixture Model) and neural network accelerator
+  - Low power keyboard and limited vocabulary recognition
+  - Up to 9.6 GMACS (16x16) of compute
+
+- Internal Memory
+
+  - 4MB shared embedded SRAM
+  - 64KB embedded SRAM for streaming samples in low power mode
+
+- External Memory Interfaces
+
+  - Up to 8MB external 16-bit PSRAM
+  - Up to 128MB external SPI flash
+
+- I/O Interfaces
+
+  - Host I/O: SPI for command and control, I2S for streaming audio, IRQ, reset, wake, optional USB 2.0 HS device
+  - Microphone: I2S/TDM 9.6 MHz max. bit clock
+  - Digital Microphone: 4 PDM ports 4.8 MHz max. bit clock
+  - Speaker: I2S/TDM 9.6 MHz max. bit clock
+  - Instrumentation: I2C master @ 100/400 MHz
+  - Debug: UART Tx/Rx/RTS/CTS up to 2.4 Mbaud/s
+  - GPIO: 10 mA sink/source, 8x PWM outputs
+
+- Power Management / Consumption
+
+  - Low power idle (memory retention); voice activity detection; play through; full active
+  - Clock and power gating support
+  - < 20 mW voice activity detection
+  - < 250 mW full active
+
+- Package (preliminary): FCCSP132 7.45 x 8.3mm 0.6/0.7mm pitch staggered/orthogonal
+
+- Temperature Range:- Commercial: 0 to 70 degree C; industrial: -40 to +85 degree C
+
+System requirements
+*******************
+
+Prerequisites
+=============
+
+The Xtensa 'toolchain'_ i.e. XCC is required to build this port. This needs a
+license and is available for Linux and Windows. Intel is currently working with Cadence
+to make the license available for wider audience. Watch this space for more updates.
```

Comment:
Given Anas's comment, maybe better than saying "Watch this space..." would be:

    For now, please contact your board supplier for licensing information and tool chain access.

Reply:
done.